test/runtests.jl

```diff
@@ -1,9 +1,15 @@
 
+using MPI
+
+MPI.Init()
 
 # unit tests
 include("unit/core/test_inputs.jl")
 
 
 # functional tests
-include("functional/test_cmdline.jl")
 include("functional/test_main.jl")
+
+MPI.Finalize()
+
```

Comment:
We should put a single line comment on why we are structuring it this way (we can only initialize MPI once per binary and the follow-up tests are piped like those in MPI.jl)